packages/monaco-editor/just.config.js

```diff
@@ -23,6 +24,7 @@ task('transform-css', transformCssTask);
 task('transform-dts', transformDtsTask);
 task('ts:esm', ts.esm);
 task('ts:commonjs', ts.commonjs);
-task('ts', parallel('ts:esm', 'ts:commonjs'));
+task('ts:postprocess', postprocessTask([...defaultLibPaths, 'esm/**/*.d.ts']));
+task('ts', series(parallel('ts:esm', 'ts:commonjs'), 'ts:postprocess'));
```

Comment:
It'd be nice if we could just redefine `ts:compile` and let the preset `ts` use the modified `ts:postprocess`. However, that won't work here since `ts:postprocess` is also redefined and the preset `ts` won't pick it up. If the task names instead worked as references somehow (scoped to a project), then projects could redefine them and still have those new definitions apply in the preset tasks.

Reply:
In this case I'm not inheriting from the basic config at all, so the task inheritance isn't so relevant, but I agree in general. (Also, this is why I wanted extensible task configs @kenotron. 😉 )